changelog/unreleased/kong/move-sockets-to-subdir.yml

```diff
@@ -0,0 +1,3 @@
+message: Moved internal Unix sockets to a subdirectory (`sockets`) of the Kong prefix.
+type: bugfix
```

Comment:
The type is `bugfix`, is it OK? I think that it would be `feat`.

Reply:
I had a similar thought: https://github.com/Kong/kong/pull/13409#discussion_r1688918468

I am fine with either bugfix or feat, though I think the former incurs less red tape with backporting to the rest of 3.x (which is planned for this change).

Reply:
bugfix seems correct to me